packages/components/src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Davatar from '@davatar/react';
+
+export interface AvatarProps {
+  /**
+   * The resolved address
+   */
+  address: string;
+  /**
+   * The size of the avatar
+   */
+  size?: number;
```

Comment:
should we try to use Chakra-UI sizes here? so that this accepts `xs`, `sm`, etc. too?